### ISSUE TITLE
`cop` report bug

### ISSUE THREAD

**Issue by benkeser** [CoVPN/correlates_reporting#311]
Getting some warnings from `sl3` calls (presumably in stochastic cop report?) and [jobs are timing out](https://travis-ci.com/github/CoVPN/correlates_reporting/jobs/505068652).

`sl3` error below

```r
In private$.train(subsetted_task, trained_sublearners) :
  Lrnr_glmnet_NULL_deviance_10_1_100_TRUE_FALSE failed with message: Error in if (any(const_vars)) { : missing value where TRUE/FALSE needed
```

**Comment by nhejazi** [comment]
This isn't really a problem, just saying that the lasso fails due to variables with constant values; `sl3` automatically drops failing learners. I'll address the timing issue by reducing the set of learners in the example SL.

**Comment by nhejazi** [comment]
Resolved by #316.
